### DISCUSSION TITLE
Conbee 2 Synology NAS

### DISCUSSION THREAD

**Discussion by mobile4** [Koenkk/zigbee2mqtt#11560] [Devices]
Please help.
I'm trying to setup zigbee2mqtt with Conbee 2 and HA.
I need detailed instruction - just started with smart home.

Hardware:
 - Synology NAS 220+
 - Conbee 2 stick
Software:
HA (virtual machine)
mosquito broker (works)
Deconz add-on. (do I need to stop it or uninstall before using zigbee2mqtt?)

It looks like zigbee2mqtt cannot find or start Conbee 2.
I already tried to add to configuration.yaml "serial: adapter: deconz" but it doesn't help. I get error after rebooting HA. Something like incorrect serial in configuration.
I checked port by this command ls -l /dev/serial/by-id - it is ttyASM0. Same port is mentioned in zigbee2mqtt config.

Here is my log:

> zigbee2mqtt@1.23.0 start
> node index.js
Zigbee2MQTT:error 2022-02-22 10:00:52: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-02-22 10:00:53: Failed to start zigbee
Zigbee2MQTT:error 2022-02-22 10:00:53: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-02-22 10:00:53: Exiting...
Zigbee2MQTT:error 2022-02-22 10:00:53: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:102:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:100:27)
    at start (/app/index.js:101:5)
[10:00:53] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.23.0 start
> node index.js

**Comment by Hedda** [comment]
> * Deconz add-on. (do I need to stop it or uninstall before using zigbee2mqtt?)

Any Zigbee Coordinator adapter can only ever be connected to one application, so yes you should uninstall DeCONZ if want to use Z2M.

If you want to run both  DeCONZ and Z2M then you will need to buy an additional Zigbee Coordinator USB adapter to use separately.

You should then just need to edit your Virtual Machine) hardware so that it passthrough the correct USB-port to the OS run in the VM. Ex:

https://community.synology.com/enu/forum/1/post/130745

**Comment by JRJSmith** [comment]
One cautionary note: when I tried running the Conbee II USB with HA running as a Synology VM, I found that the VM's connection to the USB was dropped whenever the NAS rebooted so it required action each time. I had to go into Synology VMM, select the settings for the VM, select Other tab and select the Conbee II for the USB connection. Due to this, I am setting up the Zigbee Coordinator to run on a RPi running zigbee2mqtt, continue running HA as a VM on the Synology NAS and running Mosquitto on it and pointing z2m on the RPi back to Mosquitto on the HA VM.
I have raised an issue with Synology requesting an update so that the USB connection can be made Persistent to the VM instance.
